doc/nrf/known_issues.rst

```diff
@@ -1533,7 +1552,7 @@ The combination of nRF Secure Immutable Bootloader and MCUboot fails to upgrade
 Build system
 ============
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
 
```

Comment:
Somewhere in this section add the following known issue:
```suggestion
NCSIDB-816: nanopb / protoc not specified correctly by the nRF Connect SDK Toolchain.
  nRF Connect SDK Toolchain includes nanopb / protoc when installed but the path to the tool is not propagated correctly to the nRF Connect SDK build system.
  
  ** Workaround** Find the location of the protoc executable in the nRF Connect SDK and manually its path your system's global path
```

Reply:
Thanks. Is this fixed in NCS 2.2.0? If not, I'll also add the tag for it in the PR I'll open to update main.

Reply:
No, it's not fixed in v2.2.0